addons/YarnSpinner-Godot/Runtime/DialogueRunner.cs

```diff
@@ -1291,7 +1286,7 @@ public async Task<CommandDispatchResult> DispatchCommandToNode(Command command,
     /// <inheritdoc cref="DispatchCommandToNode"/>
     /// <param name="command">The text of the command to
     /// dispatch.</param>
-    public async Task<CommandDispatchResult> DispatchCommandToNode(string command,
+    public static async Task<CommandDispatchResult> DispatchCommandToNode(string command,
```

Comment:
Technically a breaking change, but it's unlikely people were calling this directly